evennia/web/website/forms.py

```diff
@@ -0,0 +1,142 @@
+from django import forms
+from django.conf import settings
+from django.contrib.auth.forms import UserCreationForm, UsernameField
+from evennia.utils import class_from_module
+from random import choice, randint
+
+class AccountCreationForm(UserCreationForm):
+    
+    class Meta:
+        model = class_from_module(settings.BASE_ACCOUNT_TYPECLASS)
+        fields = ("username", "email")
+        field_classes = {'username': UsernameField}
+    
+    email = forms.EmailField(help_text="A valid email address. Optional; used for password resets.", required=False)
+    
+class CharacterCreationForm(forms.Form):
```

Comment:
Agreed, I've since distilled it into a more generic name/description form and intend to use this more complex form as an example of sorts. The hardest part is coming up with a sane way of letting users customize chargen forms without (a) creating and forcing users to learn yet another DSL and (b) making sure that in-game chargen adheres to the same template.